daemon/container_operations.go

```diff
@@ -1002,7 +1005,7 @@ func (daemon *Daemon) ConnectToNetwork(container *container.Container, idOrName
 			return err
 		}
 	}
-	if err := container.ToDiskLocking(); err != nil {
+	if err := container.ToDisk(); err != nil {
 		return fmt.Errorf("Error saving container to disk: %v", err)
```

Comment:
I know this is pre-existing, but should this read 
"Error saving container configuration to disk: %v"
?
(add the word configuration)